.gitmodules

```diff
@@ -11,3 +11,7 @@
 [submodule "components/mpas-source"]
 	path = components/mpas-source
 	url = git@github.com:MPAS-Dev/MPAS-Model.git
+[submodule "externals/kokkos"]
+	path = externals/kokkos
+	url = git@github.com:kokkos/kokkos.git
+	branch = develop
```

Comment:
Why develop branch, not master?

Reply:
This was a somewhat arbitrary decision by me.

Reply:
@jgfouca I remember we had to use develop when you had a PR in kokkos relative to the installation folder content, which made it into develop, but not into master. Now, however, I think we could be fine with master. There is no reason for us (I think) to chase the last available feature. Better be sure of having a stable branch, and possibly wait a few weeks for a feature. No?

Reply:
@bartgol one of the subtleties (frustrations) of submodules is that this branch= value is largely irrelevant. We can still point the submodule to any commit we want.